flex_config/utils.py

```diff
@@ -0,0 +1,16 @@
+from enum import Enum
+from typing import Any, Dict, List
+
+
+def insert_value_at_nested_key(dest_dict: Dict[str, Any], subkey_path: List[str], value: Any) -> None:
+    subkey_path = [s.lower() for s in subkey_path]
+    nested_dict = dest_dict
+    for subkey in subkey_path[:-1]:
+        nested_dict = nested_dict.setdefault(subkey, {})
+    nested_dict[subkey_path[-1]] = value
+
+
+class Environment(Enum):
+    LIVE = "live"
+    DEV = "dev"
+    SANDBOX = "sandbox"
```

Comment:
I didn't see this consumed anywhere, is this just instead of users defining it in their projects?

I would leave this out, since not every project will have these environments, especially since this is OSS.

Reply:
Yeah I was thinking that it'd be nice for flex-config to provide some handy/common enums/types/etc. to save users have to define them but I guess for now its out of scope